onnx/defs/parser.cc

```diff
@@ -434,8 +490,15 @@ Status OnnxParser::ParseSingleAttributeValue(AttributeProto& attr, AttributeProt
         attr.mutable_tp()->CopyFrom(typeProto);
       }
     } else {
-      attr.set_type(AttributeProto_AttributeType_GRAPH);
-      Parse(*attr.mutable_g());
+      if (NextIsValidFloatString()) {
+        Literal literal;
+        PARSE_TOKEN(literal);
+        attr.set_type(AttributeProto_AttributeType_FLOAT);
+        attr.set_f(static_cast<float>(std::stof(literal.value)));
+      } else {
+        attr.set_type(AttributeProto_AttributeType_GRAPH);
+        Parse(*attr.mutable_g());
```

Comment:
Can you change the `Parse` to `PARSE` ? That will check the error-status and propagate it. (This seems to be a pre-existing, undetected, bug ... I was wondering why your "not_a_good_float" doesn't produce a parse-error. I think it is due to this bug.)